docs/tour.mdx

```diff
@@ -85,21 +85,36 @@ When you are done with the tutorial, stop the runtime with `docker stop restate_
 
 To do resilience experiments later on, you can restart the runtime (without losing state) with `docker restart restate_dev`.
 
-Discover the services:
+Register the services:
+
+Now, we need to tell Restate where the service runs.
```

Comment:
```suggestion
Now, we need to tell the Restate runtime where the service runs.
```
for consistency, if you chose to pick this suggestion from #159

Reply:
We once agreed on calling the runtime just "Restate" in the docs.